scoper.inc.php

```diff
@@ -35,6 +35,15 @@
 
 return [
     'prefix' => 'Infected',
+    'expose-classes' => [
+        'Infection\Mutator\Definition',
+        'Infection\Mutator\Mutator',
+        'Infection\Mutator\MutatorCategory',
+    ],
+    'expose-namespaces' => [
+        // we have to expose it cause Mutator depends on PhpParser/Node interface, and it's not in a separate package
+        '/^PhpParser/',
```

Comment:
can be just the name:

```suggestion
        'PhpParser',
```

Reply:
This is what I started with, but running `infection.phar` then fails with:

```
PHP Fatal error:  Uncaught Error: Class "Infection\PhpParser\FileParser" not found in phar:///home/maksrafalko/apps/infection/build/infection.phar/src/Container.php:185
```

It looks like it matches `Infection\PhpParser\FileParser` in the middle